data_model/tests/data_model.rs

```diff
@@ -2,45 +2,57 @@
 
 use std::{str::FromStr, thread, time::Duration};
 
+use iroha::samples::*;
```

Comment:
seems kinds weird to include samples?

Reply:
Not particularly. It contains sample configuration files, genesis blocks and many other things that are provided *as* samples, so having them included in the tests is perfectly fine.